include/boost/histogram/detail/waldInterval.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Jay Gohil, Hans Dembinski
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_DETAIL_WALD_INTERVAL_HPP
+#define BOOST_HISTOGRAM_DETAIL_WALD_INTERVAL_HPP
+
+#include <cmath>
+
+namespace boost {
+namespace histogram {
+namespace detail {
+
+template <class T>
+T waldInterval(T n_failure_, T n_success_, T z) {
```

Comment:
snake_case

Reply:
We don't use the trailing underscore for parameters, only for private members of a class.